app/webpacker/components/RegistrationsV2/RegistrationAdministration/RegistrationAdministrationList.jsx

```diff
@@ -87,6 +87,12 @@ const partitionRegistrations = (registrations) => registrations.reduce(
   },
 );
 
+const getUserName = (registrations, userId) => {
+  const registration = registrations.find(({ user_id: id }) => id === userId);
+  if (!registration) return null;
+  return registration.user.name;
```

Comment:
What about `return registration?.user?.name`? That should also return `null` in case of doubt.